README.md

```diff
@@ -29,6 +29,10 @@ Or you can get your app name and version number such as `my-project/1.0` with
 
     \Tremby\LaravelGitVersion\GitVersionHelper::getNameAndVersion()
 
+Also, you can get the latest `commit hash` by calling this method:
+
+    \Tremby\LaravelGitVersion\GitVersionHelper::getCommitHash(8)
+
```

Comment:
No need for "commit hash" to be in backticks. That phrase is not code.

Please explain here what the `8` is, and whether it is required. If not, state what the default behaviour is.